### DISCUSSION TITLE
Add prompt reload after each batch iteration option to use batch with wildcards

### DISCUSSION THREAD

**Discussion by Anonym121** [lllyasviel/ControlNet#443] [Ideas]
Hello, for a while, I have been struggling to increase the quantity of good images I get in txt2img.
I noticed that if I use openpose, canny or other controlnet in batch using a folder with my good images, I can increase the overall quality of the new ones and I see a decrease of bad legs, arms and other bad features.
However, I extensively use wildcards to add variance and get the AI to combine ideas humans would never expect and once batch controlnet starts, it chooses a value for each wildcard and a seed, which makes homogeneous outputs.
I noticed there is a check box to increase the seed with each batch controlnet iteration. So my request is to add a checkbox or script that reloads prompt after each controlnet iteration.

This would allow making a library of poses or compositions and using it with wildcards to add different characters to the image and/or with different clothing for example.

Kindly thank you for taking your time to read this.

**Comment by geroldmeisinger** [comment]
This is something which has to been done in the UI. Automatic1111 already provides ControlNet -> Batch for example

**Comment by rpillala** [comment]
I am using SD Dynamic Prompts for wildcards. It has a checkbox in Advanced Options to unlink prompt from seed, which has allowed me to have the kind of variety you're looking for in images. The only thing is it would break reproduction of the same image using the same wildcards and seed, but if you already have the image you could use its specific prompt again. Am I understanding the issue correctly?
